lib/internal/streams/readable.js

```diff
@@ -65,7 +65,7 @@ const {
 } = require('../../ours/errors')
 const { validateObject } = require('../validators')
 const kPaused = Symbol('kPaused')
-const { StringDecoder } = require('string_decoder')
+const { StringDecoder } = require('string_decoder/')
```

Comment:
This file is copied directly from the Node sources using [build.mjs](https://github.com/nodejs/readable-stream/blob/main/build/build.mjs). As such, you shouldn't edit this file directly.

Instead, add a new replacement to the list in [replacements.mjs](https://github.com/nodejs/readable-stream/blob/fdfbf827bc29ebdcb26e93d3566891dc18f06d9e/build/replacements.mjs#L295), and then run `build.mjs` again.

Reply:
Got it - what node version should I be building with?

Reply:
Ok I've rebuilt, and also put the node version as a config option in the package.json so it is clear what node version things were built from.